astro/src/content/docs/lifecycle/authenticate-users/passwordless/magic-links.mdx

```diff
@@ -276,21 +289,19 @@ Consult the [tenant API documentation](/docs/apis/tenants) for the length limits
 
 ## Security
 
-With magic link authentication, if the user's email account is hijacked, their account on your system is compromised. However, many organizations have security policies and protections around email accounts. It is often easier to protect and regularly change one email account password than to change all of a user's passwords. Email accounts are also more likely to have two-factor authentication enabled.
+With magic link authentication, if the user's email account or phone number is hijacked, their account on your system is compromised. However, many organizations have security policies and protections around email accounts. It is often easier to protect and regularly change one email account password than to change all of a user's passwords. Email accounts are also more likely to have two-factor authentication enabled.
 
-One way to increase the security of your magic link authentications is to decrease the lifetime of the code. This will help if the email is compromised or accidentally forwarded.
+One way to increase the security of your magic link authentications is to decrease the lifetime of the code. This will help if the email or SMS message is compromised or accidentally forwarded.
 
 There are no limits on how many passwordless requests can be made for a user, but only the most recent code is valid. Using any of the others, even if they have not yet expired, will display an `Invalid login credentials` message to the user.
 
-If someone tries to log in with an email that is not present in the FusionAuth user database, they'll see the same notification as they would if the email existed. No email will be sent.
+If someone tries to log in with an email or phone number that is not present in the FusionAuth user database, they'll see the same notification as they would if the email existed (NOT TRUE FOR PHONE). No email will be sent.
```

Comment:
security note

Reply:
What does a user see with a bad phone number?

Reply:
Corrected to use our new solution.